packages/webapp/components/footer/FooterNavBar.tsx

```diff
@@ -82,7 +82,7 @@ const Tab = ({ tab, isActive }: TabProps) => {
   );
 };
 
-export function FooterNavBarV1({
+export function FooterNavBar({
```

Comment:
There are now two `FooterNavBar` component. Maybe we can rename the other one to put `Container` at the end or `Tabs` in here? Anything should work as long as they will have different names.

Reply:
Going with FooterNavBarTabs